### DISCUSSION TITLE
针对DvaJS的2个问题。

### DISCUSSION THREAD

**Discussion by liaodalin19903** [umijs/umi#10410] [Q&A] [answered]
1.DvaJS是开源的吗，GitHub中没有找到地址，没有搜索到。
2.对umi不熟悉，是否推荐dva结合umi一起使用呢？

**Comment by fz6m** [comment, marked as answer]
dva 已经不维护了，社区也不推荐使用 dva 的 sagas 实现，不要了解了。

现在管理请求状态和全局状态推荐使用 [react-query](https://umijs.org/docs/max/react-query) 、[valtio](https://umijs.org/docs/max/valtio)、[jotai](https://github.com/pmndrs/jotai) 、[zustand](https://github.com/pmndrs/zustand) 。
